docs_src/microservices/device/virtual/Ch-VirtualDevice.md

```diff
@@ -36,73 +36,73 @@ These devices are created by the virtual device service in core metadata when th
 ``` yaml
 -
   name: "Int8"
+  isHidden: false
```

Comment:
Can we just provide a links into to the profile rather than duplicate the YAML here?
ie. https://github.com/edgexfoundry/device-virtual-go/blob/v2.0.0/cmd/res/profiles/device.virtual.int.yaml#L42
and 
https://github.com/edgexfoundry/device-virtual-go/blob/v2.0.0/cmd/res/profiles/device.virtual.int.yaml#L10
for below